### DISCUSSION TITLE
最好用biliTickerBuy去抢票

### DISCUSSION THREAD

**Discussion by mikumifa** [mikumifa/cppTickerBuy#13] [通知]
cppTickerBuy很有可能在最cp30中添加验证码的功能。
验证码问题目前没办法解决（因为我必须要等到他第一次出现之后再去找应对措施，第一次出现之前没办法破解
所以第一次推荐使用biliTickerBuy （也就是b站平台的会员购的票
当然，cpp这个也可以挂着，万一没有加验证码呢

**Comment by mikumifa** [comment]
https://github.com/mikumifa/biliTickerBuy

**Comment by SkyFall5Yzy** [comment]
大佬，bili的抢票插件识别不了cpp的网址啊，这怎么弄？

**Comment by privalk** [comment]
biliTickerBuy能抢cpp的票吗

**Reply by mikumifa** [reply]
@privalk  @SkyFall5Yzy  你们误解我的意思了，我是指用b站账号去抢b站平台的票

**Comment by Stein-er** [comment]
所以今年这个问题解决了吗[doge]

**Reply by mikumifa** [reply]
去年没出现问题cpp这个脚本也能抢
